app/markets/ActivityTable.tsx

```diff
@@ -208,94 +203,94 @@ const TradeRow = ({ activity, isAIAgent }: TradeRowProps) => {
     : null;
 
   return (
-    <TableRow>
-      <TableCell className="flex items-center space-x-2 pl-4 text-text-high-em">
-        <a
-          href={getGnosisAddressExplorerLink(creatorAddress)}
-          className="hover:underline"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {shortenAddress(creatorAddress)}
-        </a>
-        {isAIAgent && <Image src="/ai.svg" alt="ai" width={18} height={18} />}
-      </TableCell>
-
-      <TableCell className="truncate text-text-high-em">
-        <div className="flex items-center space-x-2">
-          <p>
-            {activity.transactionType === TransactionType.Sell && (
-              <span className="mr-0.5 text-md font-bold">-</span>
-            )}
-            {activity.transactionType === TransactionType.Buy && (
-              <span className="mr-0.5 text-md font-bold">+</span>
-            )}
-            {formatEtherWithFixedDecimals(activity.outcomeTokensTraded)}
-          </p>
+    <RowWrapper>
+      <div className="flex items-center space-x-2">
+        <AILink address={creatorAddress} isAIAgent={isAIAgent} />
+        <div className="flex items-center space-x-1">
+          {activity.transactionType === TransactionType.Sell && (
+            <p className="text-text-low-em">sold</p>
+          )}
+          {activity.transactionType === TransactionType.Buy && (
+            <p className="text-text-low-em">bought</p>
+          )}
           <Tag
             size="xs"
             colorScheme={
               outcome ? TAG_COLOR_SCHEMES[outcome.index as 0 | 1] : 'quaternary'
             }
-            className="w-fit uppercase"
+            className="w-fit space-x-1 text-sm uppercase"
```

Comment:
I would use " " space here instead of `space-x-1`

Reply:
does not work as good, but I improved it.